docs/integrations/databases/opentelemetry/mariadb-opentelemetry.md

```diff
@@ -248,3 +248,19 @@ Use this dashboard to:
 - Examine slow query trends to determine if there are periodic performance bottlenecks in your database clusters.
 
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/MariaDB-OpenTelemetry/MariaDB-Slow-Queries.png' alt="Slow Queries" />
+
+## Create monitors for MariaDB app
+
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### MariaDB alerts
+
+| Name | Description | Alert Condition | Recover Condition |
```

Comment:
```suggestion
| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition |
```

Reply:
@amee-sumo We do not want the "Trigger Type (Critical / Warning / MissingData)" column here, as we are only showing the Alert and Recover conditions here.